x/poe/contract/download_releases.sh

```diff
@@ -8,11 +8,21 @@ if [ $# -ne 1 ]; then
 fi
 
 tag="$1"
+
 # load token from OS keychain when not set via ENV
 GITHUB_API_TOKEN=${GITHUB_API_TOKEN:-"$(security find-generic-password -a "$USER" -s "github_api_key" -w)"}
 
 rm -f version.txt
-for contract in tg4_engagement tgrade_valset tg4_mixer tg4_stake tgrade_trusted_circle tgrade_gov_reflect tgrade_oc_proposals tgrade_community_pool tgrade_validator_voting; do
+for contract in tg4_engagement tgrade_valset tg4_mixer tg4_stake tgrade_gov_reflect tgrade_community_pool tgrade_validator_voting; do
```

Comment:
`tgrade_gov_reflect` can be removed from the list.

Reply:
`tgrade_gov_reflect` is still used in system tests